pkg/merge/linstorsatelliteconfiguration.go

```diff
@@ -42,6 +44,25 @@ func SatelliteConfigurations(node *corev1.Node, configs ...piraeusv1.LinstorSate
 			storPoolMap[cfg.Spec.StoragePools[j].Name] = &cfg.Spec.StoragePools[j]
 		}
 
+		if len(cfg.Spec.PodTemplate) > 0 {
+			var u unstructured.Unstructured
+			_ = json.Unmarshal(cfg.Spec.PodTemplate, &u)
+			u.SetAPIVersion("v1")
+			u.SetKind("Pod")
+			u.SetName("satellite")
+
+			encoded, err := u.MarshalJSON()
+			if err == nil {
```

Comment:
Worthwhile at least logging an error if `err != nil`? Can't imagine why it would occur, though.